packages/docs/src/routes/integrations/menu.md

```diff
@@ -1,5 +1,12 @@
 # Integrations Guide
 
+## Integration
```

Comment:
changed order since it just felt wrong to have `deployments` before `integration` on the `/integrations` route 👼